08-communicating-results.Rmd

```diff
@@ -375,10 +378,10 @@ Reproducibility is the ability to recreate or replicate the results of a data an
 
 Reproducibility requires that we consider several key components:
 
-  * Code: The source code used for data cleaning, analysis, modeling, and reporting must be available, discoverable, documented, and shared.
-  * Data: The raw data used in the workflow must be available, discoverable, documented, and shared. If the raw data is sensitive or proprietary, we must be able to provide the data that would allow others to run our workflow.
-  * Environment: The environment of the project must be documented. Another analyst should be able to recreate the environment, including the R version, packages, operating system, and other dependencies used in the analysis.
-  * Methodology: The analysis methodology, including the rationale behind specific decisions, interpretations, and assumptions, must be documented. Others should be able to achieve the same analysis results based on the methodology report.
+  - **Code**: The source code used for data cleaning, analysis, modeling, and reporting must be available, discoverable, documented, and shared.
+  - **Data**: The raw data used in the workflow must be available, discoverable, documented, and shared. If the raw data is sensitive or proprietary, we must be able to provide the data that would allow others to run our workflow.
```

Comment:
I think we need to provide an example here.  If my data is proprietary how can I provide any data?  Maybe we can talk about RUFs or signing DUAs?